main/vlan/mvlan.pre-up

```diff
@@ -22,9 +22,11 @@ if ! ip link show "$RAW_DEVICE" >/dev/null; then
 	exit 1
 fi
 
-ip link add link "$RAW_DEVICE" name "$IFACE" type macvlan
+if ! ip link show "$IFACE" &>/dev/null; then
+	ip link add link "$RAW_DEVICE" name "$IFACE" type macvlan
+fi
 
-if ! ip link show "$IFACE" >/dev/null; then
+if ! ip link show "$IFACE" &>/dev/null; then
```

Comment:
Move to be guarded by the added if. No point to check the interface twice if it exists.